pyproject.toml

```diff
@@ -55,6 +55,9 @@ MYPYC_DEBUG_LEVEL = "0"
 # The dependencies required to build wheels with mypyc aren't specified in
 # [build-system].requires so we'll have to manage the build environment ourselves.
 PIP_NO_BUILD_ISOLATION = "no"
+# CPython 3.11 wheels aren't available for aiohttp and building a Cython extension
+# from source also doesn't work.
+AIOHTTP_NO_EXTENSIONS = "1"
```

Comment:
Should we remove from my action now so we only have 1 place with the tech debt ?

Reply:
Well the CIWB environment variables are independent from the envvars your 3.11 tox session sets. Not sure how you'd consolidate them. I have not configured cibuildwheel to use tox as part of the wheel test phase because it would it'd be fairly involved.